pymatgen/analysis/reaction_rates.py

```diff
@@ -0,0 +1,566 @@
+    and products) and which can, from the energetics of those individual molecules, predict the
+    rate constant, rate law, and thus the chemical kinetics of the reaction.
+
+    NOTE: It is assumed that only one transition state is present.
+
+    Args:
+        reactants (list): list of MoleculeEntry objects
+        products (list): list of MoleculeEntry objects
+        transition_state (MoleculeEntry): MoleculeEntry representing the transition state between
+            the reactants and the products
+        reaction (dict, or None): optional. If None (default), the "reactants" and
+        "products" lists will serve as the basis for a Reaction object which represents the
+        balanced stoichiometric reaction. Otherwise, this dict will show the number of molecules
+        present in the reaction for each reactant and each product in the reaction.
+
+    Returns:
+        None
+    """
+
+    def __init__(self, reactants, products, transition_state, reaction=None):
+        """
+
+        """
+
+        self.reactants = reactants
+        self.products = products
+        self.transition_state = transition_state
+
+        if reaction is None:
+            rct_mols = [r.mol_graph.molecule for r in self.reactants]
+            pro_mols = [p.mol_graph.molecule for p in self.products]
+            try:
+                self.reaction = self.generate_reaction(rct_mols, pro_mols)
+            except ReactionError:
+                # Reaction cannot be balanced
+                self.reaction = None
+        else:
+            self.reaction = reaction
+
+    @classmethod
+    def generate_reaction(cls, rct_mols, pro_mols):
+        """
+        Generate a pymatgen Reaction object, given lists of reactant and product Molecule objects.
+
+        Args:
+            rct_mols (list): list of Molecule objects representing reactants
+            pro_mols (list): list of Molecule objects representing products
+
+        Note: if reaction cannot be balanced, this method will raise a ReactionError
+
+        Returns:
+            reaction (Reaction): pymatgen Reaction object representing the reaction between the
+                reactants and products.
+        """
+
+        rct_comps = [r.composition for r in rct_mols]
+        pro_comps = [p.composition for p in pro_mols]
+
+        return Reaction(rct_comps, pro_comps)
+
+    @property
+    def net_energy(self):
+        """
+        Calculate net reaction energy (in Hartree).
+        """
+        rct_energies = [r.energy for r in self.reactants]
+        pro_energies = [p.energy for p in self.products]
+
+        return sum(pro_energies) - sum(rct_energies)
+
+    @property
+    def net_enthalpy(self):
+        """
+        Calculate net reaction enthalpy (in kcal/mol).
+        """
+        rct_enthalpies = [r.enthalpy for r in self.reactants]
+        pro_enthalpies = [p.enthalpy for p in self.products]
+
+        return sum(pro_enthalpies) - sum(rct_enthalpies)
```

Comment:
I believe that this should also include deltaE since the enthalpy reported by Q-Chem (and thus how we use it here) doesn't include the electronic energy. Check out how I calculate the free energy in mol_entry.py